examples/tendermint/script/src/main.rs

```diff
@@ -1,50 +1,32 @@
 use sp1_sdk::SP1ProofWithPublicValues;
 use std::time::Duration;
-use tokio::runtime::Runtime;
 
-use reqwest::Client;
 use sp1_sdk::{utils, ProverClient, SP1Stdin};
 
-use tendermint_light_client_verifier::options::Options;
-use tendermint_light_client_verifier::types::LightBlock;
-use tendermint_light_client_verifier::ProdVerifier;
-use tendermint_light_client_verifier::Verdict;
-use tendermint_light_client_verifier::Verifier;
+use tendermint_light_client_verifier::{
+    options::Options, types::LightBlock, ProdVerifier, Verdict, Verifier,
+};
 
-use crate::util::fetch_latest_commit;
-use crate::util::fetch_light_block;
+use crate::util::load_light_block;
 
 const TENDERMINT_ELF: &[u8] = include_bytes!("../../program/elf/riscv32im-succinct-zkvm-elf");
+
 mod util;
 
-async fn get_light_blocks() -> (LightBlock, LightBlock) {
-    // Uniquely identify a peer in the network.
-    let peer_id: [u8; 20] = [
-        0x72, 0x6b, 0xc8, 0xd2, 0x60, 0x38, 0x7c, 0xf5, 0x6e, 0xcf, 0xad, 0x3a, 0x6b, 0xf6, 0xfe,
-        0xcd, 0x90, 0x3e, 0x18, 0xa2,
-    ];
-    const BASE_URL: &str = "https://public-celestia-rpc.numia.xyz/";
-    let client = Client::new();
-    let url = format!("{}/commit", BASE_URL);
-    let latest_commit = fetch_latest_commit(&client, &url).await.unwrap();
-    let block: u64 = latest_commit.result.signed_header.header.height.into();
-    println!("Latest block: {}", block);
-    let light_block_1 = fetch_light_block(2095390, peer_id, BASE_URL)
-        .await
-        .expect("Failed to generate light block 1");
-    let light_block_2 = fetch_light_block(2095392, peer_id, BASE_URL)
-        .await
-        .expect("Failed to generate light block 2");
+// Loads light blocks from the `files` subdirectory
```

Comment:
this comment is bad

1) use /// not // for functions
2) use proper punctuation